youtube_dl_gui/optionsframe.py

```diff
@@ -281,7 +281,8 @@ class GeneralTab(TabPanel):
         ('pt_BR', 'Portuguese'),
         ('ru_RU', 'Russian'),
         ('es_ES', 'Spanish'),
-        ('tr_TR', 'Turkish')
+        ('tr_TR', 'Turkish'),
+        ('zh_CN', 'Chinese')
```

Comment:
You should sort alphabetically as suggested in the [help section of the guide](https://github.com/MrS0m30n3/youtube-dl-gui/blob/new_layout/locale_build/HOWTO.md#help) (**make sure to sort alphabetically based on the language name**).